lib/js/src/streaming/video/_VideoStreamingParameters.js

```diff
@@ -104,6 +104,10 @@ class _VideoStreamingParameters {
         if (formats !== null && formats.length > 0) {
             this._format = formats[0];
         }
+        const fps = capability.getPreferredFPS();
+        if (fps != null) {
```

Comment:
```suggestion
        if (fps !== null) {
```
Linter rule requires use of `!==`

Reply:
Sorry for the typo. Updated.